filebeat/docs/docker-input.asciidoc

```diff
@@ -0,0 +1,64 @@
+:type: docker
+
+[[docker-input]]
+=== Docker input
```

Comment:
Hmmm... docker input/prospector is available since 6.1. Only found it [mentioned here](https://www.elastic.co/guide/en/beats/filebeat/current/configuration-filebeat-options.html#_literal_type_literal).

As this is a new file, maybe we want to have an extra PR for it? 

Maybe we want to have a separate doc per input type? 

How about naming the file `input-docker.asciidoc`. This way, all input related docs will be displayed together in the file browser.

With inputs/modules being mostly self-contained, I wonder if we want to add the docs to the modules/inputs `_meta` path.

Reply:
@urso 

- re docker prospector for 6.1: Getting the prospector options broken into separate topics has been on my to-do list for awhile. I probably should have done a separate PR for the structural changes to make it easier to backport them. But since I didn't, I'll just have to create a separate PR for 6.1/6.2 that has relevant changes (if we decide to backport). First I want to make sure this gets into 6.3.
- This PR does add a separate doc for each input.
- I'll rename the files as you've suggested
- If we use _meta folders, I'd like to keep it simple and add an include rather than doing a copy to the docs dir. I've asked ph for his input.

Reply:
I confirmed that the docker options are covered in 6.1, just not ideally organized. Backporting this work to 6.2 would be a pretty big effort. The reorg isn't straightforward because it involves moving a lot of content around. I'll revisit the decision later if it makes sense, but for now, I'd like to keep this in 6.3 and not backport.